net-analyzer/zeek/zeek-3.0.1.ebuild

```diff
@@ -0,0 +1,124 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{6,7,8} )
+inherit cmake-utils python-single-r1
+
+DESCRIPTION="The Zeek Network Security Monitor"
+HOMEPAGE="https://www.zeek.org"
+SRC_URI="https://www.zeek.org/downloads/${P}.tar.gz"
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="curl debug geoipv2 ipsumdump ipv6 jemalloc kerberos +python sendmail \
+	static-libs tcmalloc +tools +zeekctl"
+
+RDEPEND=">=sys-libs/glibc-2.10
+	curl? ( net-misc/curl )
+	dev-libs/actor-framework
+	dev-libs/openssl:0
+	geoipv2? ( dev-libs/libmaxminddb )
+	ipsumdump? ( net-analyzer/ipsumdump[ipv6?] )
+	jemalloc? ( dev-libs/jemalloc )
+	kerberos? ( virtual/krb5 )
+	net-libs/libpcap
+	python? ( ${PYTHON_DEPS}
+		dev-python/pybind11[${PYTHON_USEDEP}] )
+	sendmail? ( virtual/mta )
+	sys-libs/zlib
```

Comment:
Please sort that unconditional deps first, then the ones with use flag conditions.

Reply:
Done.